### PR TITLE
Lmb 826 make add message optional in change modal

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## Description

The modal that adds a description to your change does not need to be shown on the legislatuur page.

## How to test

Correct a mandataris. The modal should show
Edit a mandataris in the IV page the modal should not show.

### CODE REVIEW

File: app/components/mandatarissen/mandataris-extra-info-card.hbs [hunk 1]
```diff
@@ -43,6 +43,7 @@
 >
   <div class="au-o-box">
     <Form::Instance
+      @showHistoryDescription={{true}}
```
Comment (Rahien) [inline app/components/mandatarissen/mandataris-extra-info-card.hbs:46]: i would make true the default

File: app/components/form/save-with-history.hbs [hunk 2]
```diff
@@ -1,30 +1,36 @@
...
-          @loadingMessage="Aan het opslaan"
-        >
-          Bevestig
-        </AuButton>
-      </Group>
-    </AuToolbar>
-  </Modal.Footer>
-</AuModal>
\ No newline at end of file
+{{#if @isModalOpen}}
```
Comment (Rahien) [inline app/components/form/save-with-history.hbs:1]: this if seems redundant?

File: app/components/form/save-with-history.hbs [hunk 3]
```diff
@@ -1,30 +1,36 @@
...
+              @loadingMessage="Aan het opslaan"
+            >
+              Bevestig
+            </AuButton>
+          </Group>
+        </AuToolbar>
+      </Modal.Footer>
+    </AuModal>
+  {{else}}
+    <code {{did-insert @save}} />
```
Comment (Rahien) [inline app/components/form/save-with-history.hbs:34]: this doesn't feel quite right. Shouldn't the instance component just do this? and either trigger the saveWithHistory or not depending on @showHistoryDescription?

**Review by Rahien — CHANGES_REQUESTED** [review]

File: app/components/mandatarissen/mandataris-extra-info-card.hbs (hunk 1 above)
Comment (JonasVanHoof) [reply app/components/mandatarissen/mandataris-extra-info-card.hbs:46]: I actually thought you would say it ;p I'll change it!

File: app/components/form/save-with-history.hbs (hunk 3 above)
Comment (JonasVanHoof) [reply app/components/form/save-with-history.hbs:34]: Ill update it that it is being handled  in a higher layer

**Review by Rahien — APPROVED** [review]

**Pull request merged**